zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -312,7 +308,8 @@ public boolean runParagraph(String noteId,
     p.setConfig(config);
 
     if (note.isPersonalizedMode()) {
-      p = note.getParagraph(paragraphId);
+//      p = note.getParagraph(paragraphId);
+      p = p.getUserParagraph(context.getAutheInfo().getUser());//fix for ZEPPELIN-3065 -- shoaib.rehman
```

Comment:
`//fix for ZEPPELIN-3065 -- shoaib.rehman`
I think this comment can be deleted because it doesn't make sense.

Reply:
@srehmann , Please delete `//      p = note.getParagraph(paragraphId);`